test/tools/ossfuzz/yulProto_diff_ossfuzz.cpp

```diff
@@ -78,12 +78,8 @@ DEFINE_PROTO_FUZZER(Program const& _input)
 	);
 
 	// Parse protobuf mutated YUL code
-	if (
-		!stack.parseAndAnalyze("source", yul_source) ||
-		!stack.parserResult()->code ||
-		!stack.parserResult()->analysisInfo ||
-		!Error::containsOnlyWarnings(stack.errors())
-	)
+	if (!stack.parseAndAnalyze("source", yul_source) || !stack.parserResult()->code ||
+		!stack.parserResult()->analysisInfo || !Error::containsOnlyWarnings(stack.errors()))
```

Comment:
I'm just randomly looking at changes :-). But this for example seems to still be a gross violation of the current coding style...

Reply:
Yeah, the original version was much easier to parse visually. If breaking after a parenthesis is a problem then I think at least something like this would be preferable (though still a downgrade):

```c++
	if (!stack.parseAndAnalyze("source", yul_source) ||
		!stack.parserResult()->code ||
		!stack.parserResult()->analysisInfo ||
		!Error::containsOnlyWarnings(stack.errors())
	)
```